.github/workflows/main.yml

```diff
@@ -166,7 +166,7 @@ jobs:
           name: android-quality-reports
 
       - name: SonarQube Analysis
-        run: ./gradlew sonarqube
+        run: ./gradlew clean sonarqube --stacktrace
```

Comment:
The download tasks above are providing coverage/test data to SonarQube.
So, you should not clean the build

Reply:
The idea is to try a clean then normal run to see if it would resolve the issue. I wasn't able to find a solution so far, so I am trying some things...